news/2022-08-03-new-featured-artist-hamu.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Hamu"
+date: 2022-08-03 11:00:00 +0000
+---
+
+Experience a new world of emotive tracks with our latest Featured Artist, **Hamu!**
+
+![](https://assets.ppy.sh/artists/289/header.jpg)
+
+Founder of the music label *Foxtail-Grass Studio*, you might already be familiar with some of [**Hamu**](https://osu.ppy.sh/beatmaps/artists/289)'s work if you're a fan of the Touhou Project and its sprawling catalogue of fan rearrangements. Beyond that, if you've ever watched a stream from the tremendously popular Virtual YouTuber talents of *Shirakami Fubuki*, *Sakura Miko* and *Minato Aqua* from *Hololive Production*, you've likely heard some of his BGM tracks before, not to mention his role as the producer for a few of the aforementioned's original songs.
+
+With an expansive and emotive sound, we're absolutely confident that you'll love what **Hamu** has to offer, so we've partnered up to snag **16** new tracks across his entire range, from slow meandering through moonlit vistas to head-bopping, high-energy J-Pop that'll get stuck in your head for the rest of all eternity.
+
+Thanks to the crowned creators of the [Mappers' Guild](https://mappersguild.com/), we're also excited to showcase an all-new beatmap to get things rolling, played by esteemed osu! veterans [Mlaw](https://osu.ppy.sh/users/3126596) and corn connoisseur [Woey](https://osu.ppy.sh/users/3792472) IRL at COE2022:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/289/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Lemrina
+
+Check out [the collaborative set featured in the video above](https://osu.ppy.sh/beatmapsets/1799400) hosted by [Usaha](https://osu.ppy.sh/users/6443117) of the Mappers' Guild!
```

Comment:
umm i checked http://assets.ppy.sh/artists/289/release_showcase.mp4 and the song there is "Hamu - Secret Clover" (http://osu.ppy.sh/s/1815789) instead of "Lemrina" apparently 😦

Reply:
I am brain damaged. Good catch.